PurchasesTests/Purchasing/PurchasesTests.swift

```diff
@@ -2802,6 +2809,7 @@ class PurchasesTests: XCTestCase {
         }
     }
 
+    @available(iOS, deprecated: 9.0) // Ignore deprecation warnings
     func testSetDebugLogsEnabledSetsTheCorrectValue() {
```

Comment:
interesting, we should probably apply this in a bunch of places to get rid of the warnings. perhaps as a separate PR? 
I'd probably go with a value like `0.1` so it's super obvious that it's not tied to the OS version. we can even tie it to the swift version instead so that it's not even OS-specific, since otherwise I think we might still get the warnings when compiling tests for like tvOS

Reply:
✅